samples/tab-deeplink/csharp/DeepLinkBot/Views/Home/ExtendedDeepLinks.cshtml

```diff
@@ -226,10 +157,87 @@ microsoftTeams.app.openLink("https://teams.microsoft.com/l/app/1542629c-01b3-4a6
                             </pre>
                         </div>
                     </div>
-                    <button 
-                        class="callDeeplinkButton"
-                        onclick="onAppInstallDialogClick()"
-                    >
+                    <button class="callDeeplinkButton"
+                            onclick="onAppInstallDialogClick()">
+                        Open
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="card-container-div">
+            <div>
+                <div class="card">
+                    <div class="card-body">
+                        <div class="card-header">
+                            <b>Start new chat</b>
+                        </div>
+                        <div class="configure-header">
+                            To use this option, configure the function using following
+                            parameters
+                            <!-- Since we are using pre tag for having the code snippet in specified format ignoring the indentation here -->
+                            <pre>
+const chatPromise = chat.openGroupChat({ users: [user] });
+    chatPromise.
+        then((result) => { console.log(result) }).
+        catch((error) => { console.log(error) });
+                            </pre>
+                        </div>
+                    </div>
+                    <button class="callDeeplinkButton"
+                            onclick="startNewChat()">
+                        Open
+                    </button>
+                </div>
+            </div>
+            <div>
+                <div class="card">
+                    <div class="card-body">
+                        <div class="card-header">
+                            <b>Navigate to chat (Personal chat with bot)</b>
+                        </div>
+                        <div class="configure-header">
+                            To use this option, configure the function using following
+                            parameters
+                            <!-- Since we are using pre tag for having the code snippet in specified format ignoring the indentation here -->
+                            <pre>
+microsoftTeams.app.openLink(`https://teams.microsoft.com/l/entity/f6661eaa-0cf9-4eb2-93b1-58278abaf4d4/conversations`);
```

Comment:
What is this? f6661eaa-0cf9-4eb2-93b1-58278abaf4d4

Hardcoded value?

Please check in nodejs sample as well.

Reply:
Yes removed it